mitodl/mitxonline
# adding enrollment change tracking

### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/3580

### Description (What does it do?)
Track enrollment change data by adding call stack info to a field "modified_by" to the audit model.


### How can this be tested?
Create an enrollment for user then try modifying it through running a task or calling some of the methods that modify an enrollment from django shell:
`deactivate_run_enrollment(enrollment, "unenrolled")`

Then go to the admin and view the "CourseEunEnrollmentAudit" it should look something like this:

<img width="450" alt="Screenshot 2024-02-29 at 10 48 12 AM" src="https://github.com/mitodl/mitxonline/assets/7574259/3d773857-2702-4098-b07d-b054713dcf96">
